pkg/mtu/mtu.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Authors of Cilium
```

Comment:
Can you provide a `doc.go` file inside of this package? See other packages for an example of this.

Reply:
Do we generate docs from this somewhere? Or is it automatically integrated with something like goland?

Reply:
I'm not sure. I just know this is a convention that we follow - @tgraf has asked for it in the past.